zimui/src/components/channel/tabs/PlaylistsTab.vue

```diff
@@ -8,7 +8,8 @@ import PlaylistGrid from '@/components/playlist/PlaylistGrid.vue'
 import TabInfo from '@/components/common/ViewInfo.vue'
 
 const main = useMainStore()
-const playlists = ref<PlaylistPreview[]>()
+const playlists = ref<PlaylistPreview[]>([])
+const isLoading = computed(() => playlists.value.length === 0)
```

Comment:
Wouldn't this cause an issue if for some reason we load this component with an empty playlist list? Wouldn't it make more sense to set this to `undefined` when it is not yet loaded? I'm not sure about the impact, but I find this pretty fragile.

Or maybe even better, we could have a ref `loading` which is true by default and set to false in `fetchData`?

Reply:
Sure, changed it to a `ref` in [d1930a0](https://github.com/openzim/youtube/pull/289/commits/d1930a05040e8a51b581c2be289d366f87a93806)